### PR TITLE
RSDK-894 - Remove log for already registered camera streams

### PR DESCRIPTION
**Pull request by SeanAvery**
### Summary
This PR removes Warn log for already registered camera/audio streams.

```
$ go run web/cmd/server/main.go -config etc/configs/fake.json
```

Make change to `fake.json` -> no camera/audio "already registered" log :)

### CODE REVIEW

File: robot/web/web.go [hunk 1]
```diff
@@ -486,7 +486,7 @@ func (svc *webService) addNewStreams(ctx context.Context) error {
 		// Skip if stream is already registered, otherwise raise any other errors
 		var registeredError *gostream.StreamAlreadyRegisteredError
 		if errors.As(err, &registeredError) {
-			svc.logger.Warn(registeredError.Error())
```
[the remark this answers is not in the document]
Comment (SeanAvery) [reply robot/web/web.go:489]: sounds good - simply removed log.

**Review by bhaney — APPROVED** [review]
Congrats on your first PR at Viam

**Review by edaniels — APPROVED** [review]
LGTM. 100% deletion

**Review by bazile-clyde — APPROVED** [review]
LGTM! You're now an official contributor. Congrats!

**Pull request merged**